src/VocabularyManager.Api/Controllers/WordListController.cs

```diff
@@ -0,0 +1,140 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using VocabularyManager.Core.Entities;
+using VocabularyManager.Core.Interfaces;
+using VocabularyManager.Infrastructure.Validators;
+using VocabularyManager.Infrastructure.Exceptions;
+using VocabularyManager.UseCases.DTOs;
+using FluentValidation;
+
+namespace VocabularyManager.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class WordListController : ControllerBase
```

Comment:
I mean dictionary under the word list, so word list just can contain own data and list of words. does it sound more natural as dictionary?

Reply:
it still difficult for me to understand what they do.
If it would be endpoint of `/dictionaries` I would thought about them as `key, pair` value

Can you explain me functionallity on both?